src/tests/unittests/utils/test_serialization.py

```diff
@@ -137,3 +134,29 @@ def test_unserialize(self):
         data = b'{"hello":"world"}'
         unserialized = unserialize(data)
         self.assertDictEqual(unserialized, {'hello': 'world'})
+
+    def test_regression_serialize_bytes(self):
+        serialized = serializer.serialize(b'\x00\x00\x00\x00\x00\x00\xf0?\x00\x00')
```

Comment:
nit: should we assign this bytestring to a local variable to make it more immediately obvious that this is the same string as the expected string from assertEqual?

Reply:
Normally I would but also this test was copied from the other Pull request. I will change this in follow up work on qilib